### PR TITLE
Update README with Service Binding community meeting and mailing list details

### PR DESCRIPTION
**Pull request by DhritiShikhar**
### Jira Issue

https://issues.redhat.com/browse/APPSVC-627

### Motivation

[1] Adds details about service binding community meeting in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,11 @@ The following example scenarios are available:
 [Binding an Imported app to an Off-cluster Operator Managed IBM Cloud Service](examples/nodejs_ibmcloud_operator/README.md)
 
 [Binding an Imported app in one namespace with an In-cluster Managed PostgreSQL Database in another namespace](examples/nodejs_postgresql_namespaces/README.md)
+
+## Service Binding Meetings
```
Comment (Avni-Sharma) [inline README.md:99]: We can rename this to Community, discussion, contribution, and support, something more descriptive.

And we should also  add the mailing list that was created for any support/discussion

Comment (DhritiShikhar) [reply README.md:99]: @Avni-Sharma Let's rename the heading to `Service Binding Community`?

Comment (pedjak) [reply README.md:99]: This is README of Service Binding project already, hence no need to repeat "Service Binding", agree with @Avni-Sharma . +1 for  "Community, discussion, contribution, and support" .

Comment (DhritiShikhar) [reply README.md:99]: @pedjak What about mailing list info?
Should it be here or not?

Comment (pedjak) [reply README.md]: good idea, +1.

Comment (DhritiShikhar) [reply README.md]: Done

**Comment by DhritiShikhar** [comment]
@pedjak Thanks for approval. 
Updated commit message.
Is this good to be merged now?

**Pull request merged**